docker-compose.override.yml

```diff
@@ -18,4 +18,7 @@ services:
   lantern-mq:
     ports:
       - "15672:15672"
-      - "5672:5672"
\ No newline at end of file
+      - "5672:5672"
+  lantern_front_end:
+    ports:
```

Comment:
Can remove the lantern_front_end entry from the override since it is duplicating what is in the docker-compose.

Reply:
👍